driver/mounter.go

```diff
@@ -208,6 +214,50 @@ func (m *mounter) Unmount(target string) error {
 	return mount.CleanupMountPoint(target, m.kMounter, true)
 }
 
+func (m *mounter) IsRunning(source string, cmdContext execContext) bool {
+	cmd := fmt.Sprintf("ls -l %s | awk '{print $NF}'", source)
+	out, err := cmdContext("/bin/sh", "-c", cmd, source).CombinedOutput()
+
+	if err != nil {
+		m.log.WithFields(logrus.Fields{
+			"cmd": cmd,
+		}).Errorf("error retrieving the mount name, err: %v", err)
+
+		return false
+	}
+
+	mountName := strings.TrimSuffix(string(bytes.Replace(out, []byte("../../"), []byte(""), 1)), "\n")
+
+	cmd = fmt.Sprintf("cat /sys/class/block/%s/device/state | grep -x running | wc -l", mountName)
```

Comment:
I'd prefer if we read the file into memory and parsed it for the `running` field.